### DISCUSSION TITLE
Question: transferring data between environments

### DISCUSSION THREAD

**Discussion by tibotiber** [hasura/graphql-engine#6837] [Question]
Hi, do you have any prescripted method to transfer data from my dev database to my prod database? It's not for the whole database, but cherry-picked data. Ideally, I would imagine selecting specific rows of a table, export it as a SQL file that can be used as a migration (upsert statements). Alternatively, there is already a JSON/CSV export, if we could manually import from these formats, It'd already be great. Thanks in advance.

**Comment by dgchrt** [comment]
The `Export data` button already exists, now we just need the `Import data` button. 🙏
